### ISSUE TITLE
[Feature request] Adding the --start flag

### ISSUE THREAD

**Issue by mnazaal** [kabouzeid/turm#34]
According to the [`squeue` documentation](https://slurm.schedmd.com/squeue.html#OPT_start) the  `--start` flag would show expected start times for jobs, which is helpful for users to set time related expectations regarding queued jobs.

**Comment by kabouzeid** [comment]
Good point. I don't have time to implement this, but if someone wants to create a PR for this, you would need to add `StartTime` in the `--Format` args here:

https://github.com/kabouzeid/turm/blob/b2edadc32adc1730b98f983e35ef85f974f5e565/src/job_watcher.rs#L46
